src/importexport/tabledit/internal/importtef.cpp

```diff
@@ -0,0 +1,1045 @@
+        && tefReadingList.at(0).firstMeasure == 1 && tefReadingList.at(0).lastMeasure == static_cast<int>(tefMeasures.size())
+        && tefReadingList.at(1).firstMeasure == 1 && tefReadingList.at(1).lastMeasure == static_cast<int>(tefMeasures.size())
+        ) {
+        LOGD("do it");
+        if (score->measures()->empty()) {
+            LOGE("no measures in score");
+            return;
+        }
+        Measure* first { score->firstMeasure() };
+        Measure* last { score->lastMeasure() };
+        first->setRepeatStart(true);
+        last->setRepeatEnd(true);
+    }
+    else {
+        LOGD("no score repeat");
+    }
+}
+
+void TablEdit::createScore()
+{
+    createProperties();
+    createParts();
+    createTitleFrame();
+    createMeasures();
+    createNotesFrame();
+    createContents();
+    createRepeats();
+    createTexts();
+    createLinkedTabs();
+}
+
+void TablEdit::createTempo()
+{
+    mu::engraving::Measure* measure = score->firstMeasure();
+    mu::engraving::Segment* segment = measure->getSegment(mu::engraving::SegmentType::ChordRest, mu::engraving::Fraction(0, 1));
+    mu::engraving::TempoText* tt = new mu::engraving::TempoText(segment);
+    tt->setTempo(double(tefHeader.tempo) / 60.0);
+    tt->setTrack(0);
+    tt->setFollowText(true);
+    muse::String tempoText = mu::engraving::TempoText::duration2tempoTextString(mu::engraving::DurationType::V_QUARTER);
+    tempoText += u" = ";
+    tempoText += muse::String::number(tefHeader.tempo);
+    tt->setXmlText(tempoText);
+    segment->add(tt);
+}
+
+void TablEdit::createTexts()
+{
+    for (const auto& textMarker : tefTextMarkers) {
+        LOGD("position %d string %d text marker %d", textMarker.position, textMarker.string, textMarker.index);
+        bool ok { true };
+        const auto part = partIdx(textMarker.string, ok);
+        if (!ok) {
+            LOGD("error: invalid string %d", textMarker.string);
+            continue;
+        }
+        const auto track = part * VOICES;
+        LOGD("part %zu track %zu", part, track);
+
+        Fraction tick { textMarker.position, 64 }; // position is in 64th
+        Measure* measure { tick2measure(score, tick) };
+        if (!measure) {
+            LOGD("error: no measure");
+            continue;
+        }
+        else {
+            LOGD("measure %p", measure);
+        }
+        Segment* segment { measure->getSegment(mu::engraving::SegmentType::ChordRest, tick) };
+        if (!segment) {
+            LOGD("error: no segment");
+            continue;
+        }
+
+        StaffText* staffText = Factory::createStaffText(segment);
+        muse::String text { tefTexts.at(textMarker.index).c_str() };
+        staffText->setPlainText(text);
+        staffText->setTrack(track);
+        segment->add(staffText);
+    }
+}
+
+void TablEdit::createTitleFrame()
+{
+    VBox* vbox = Factory::createTitleVBox(score->dummy()->system());
+    vbox->setTick(mu::engraving::Fraction(0, 1));
```

Comment:
Perhaps you need to call `vbox->manageExclusionFromParts(/*exclude*/ false);`, like in `MasterNotation::applyOptions` for new scores.